themes/default/content/templates/static-website/aws/index.md

```diff
@@ -0,0 +1,802 @@
+---
+title: AWS Static Website
+meta_desc: The AWS Static Website template makes it easy to deploy a static website on AWS with Pulumi, Amazon S3, and Amazon CloudFront.
+meta_image: meta.png
+card_desc: Deploy a static website on AWS with Pulumi, Amazon S3, and Amazon CloudFront.
+layout: template
+template:
+    prefix: static-website-aws
+cloud:
+  name: Amazon Web Services
+  slug: aws
+---
+
+The AWS Static Website template deploys an HTML-based website on AWS with Pulumi. It uses an [Amazon S3 bucket]({{< relref "/registry/packages/aws/api-docs/s3/bucket" >}}) for file storage, configures the bucket to host a website, and deploys an [Amazon CloudFront Distribution]({{< relref "/registry/packages/aws/api-docs/cloudfront/distribution" >}}) to serve the website with low latency, caching, and HTTPS. The template generates a complete Pulumi program, including placeholder web content, to give you working project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
```suggestion
The AWS Static Website template deploys an HTML-based website on AWS with Pulumi. It uses an [Amazon S3 bucket]({{< relref "/registry/packages/aws/api-docs/s3/bucket" >}}) for file storage, configures the bucket to host a website, and deploys an [Amazon CloudFront Distribution]({{< relref "/registry/packages/aws/api-docs/cloudfront/distribution" >}}) to serve the website with low latency, caching, and HTTPS. The template generates a complete Pulumi program, including placeholder web content, to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Reply:
in the preview, the links to the API docs aren't working for me

Reply:
These won't work in previews, yeah, since the API does pages aren't built with PR previews. They'll work when we merge the PR and release the site, though.